cmd/internal/flowmode/cmd_convert.go

```diff
@@ -174,3 +180,15 @@ func supportedFormatsList() string {
 	}
 	return strings.Join(ret, ", ")
 }
+
+func parseExtraArgs(extraArgs string) []string {
+	var result []string
+	if extraArgs != "" {
+		arguments := strings.Fields(extraArgs)
+		for _, arg := range arguments {
+			parts := strings.Split(arg, "=")
+			result = append(result, parts...)
+		}
```

Comment:
Hmm, what if someone passes their extra args using another (valid in Cobra library we use) syntax? I think the following are all valid ways: `--key=value`, `--key value`, `-k=value` (when there is shorthand defined), `-k value`, `--boolVariable` (makes the boolVariable True). 

I wonder if we can use Cobra library to somehow do the parsing instead?
Also a test for this function would be a good idea.

Reply:
Took a closer look at this and we don't use Cobra in static mode. Also I think this should handle most cases other than trying to pass `-something="key=value,key2=value2"`, but perhaps that's too much of an edge case?

Reply:
It's a good point though to look closer and add some tests so it's clear how this works and what it doesn't work with. I can spend some time trying to clean up the implementation on this one since it is a bit brittle as implemented.

Reply:
I'll take a look tomorrow about tidying it up and adding some fail test cases, but what do you think about the new direction for parsing arguments?

Reply:
That will do it! thanks :)